web/index.html

```diff
@@ -22,13 +22,11 @@
     <link rel="stylesheet" href="packages/devtools/debugger/debugger.css">
     <link rel="stylesheet" href="packages/devtools/logging/logging.css">
 
-    <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
```

Comment:
We weren't actually using the charts .js library, correct?

Reply:
Yeah, this was from a while ago when charts were displayed via google charts. We don't anymore - we just use straight svg.